datalad/support/gitrepo.py

```diff
@@ -1021,15 +1021,10 @@ def configure_fake_dates(self):
     def fake_dates_enabled(self):
         """Is the repository configured to use fake dates?
         """
-        if self._fake_dates_enabled:
-            return True
-
         if self._fake_dates_enabled is None:
-            fake = self.config.get('datalad.dataset.fakedates')
-            if fake == "true":
-                self._fake_dates_enabled = True
-                return True
-        return False
+            self._fake_dates_enabled = \
+                self.config.getbool('datalad.dataset', 'fakedates', default=False)
+        return self._fake_dates_enabled
```

Comment:
Doh, thanks.  My intention was to store `self._fake_dates_enabled = False`, but as you note, that wasn't actually happening.

And thanks for making the bool check consistent with the rest of datalad (though I don't see that advantage of datalad allowing multiple values here rather than git's convention of "true").

Reply:
re advantages -- probably none.  Just elderlies forgetting their bools ;)